daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -39,15 +39,21 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
     val internedDottedNames =
       decodeInternedDottedNames(lfPackage.getInternedDottedNamesList.asScala, internedStrings)
 
+    val dependencyTracker = new PackageDependencyTracker(packageId)
+
     Package(
-      lfPackage.getModulesList.asScala
+      modules = lfPackage.getModulesList.asScala
         .map(
           ModuleDecoder(
```

Comment:
Very minor.  `ModuleDecoder` constructor starts to have to many arguments for my little brain. I would consider using parameter names, when it is called.